steveify1/tree-stack
# #15 feature(home-page):change banner illustration

#### What does this PR do?
Change the svg illustration on the banner of the home page
#### Description of Task to be completed?
- replace the svg illustration on the homepage with a rocket
illustraion
#### How should this be manually tested?
- clone the repository.
- cd into the project  directory.
- run `npm install` to install all the project's dependencies.
- run `npm start` to start the development server.
- visit `http://localhost:3000` on your browser to view the website. At this point, you should be seeing the homepage
#### Any background context you want to provide?
n/a
#### What are the relevant pivotal tracker stories?
#15 
#### Screenshots (if appropriate)
#### Questions:
